Lib/sqlite3/test/userfunctions.py

```diff
@@ -168,6 +170,7 @@ def setUp(self):
         self.con = sqlite.connect(":memory:")
 
         self.con.create_function("returntext", 0, func_returntext)
+        self.con.create_function("returntextwithnull", 0, func_returntextwithnull)
```

Comment:
Add also similar test for `create_aggregate()`.

Reply:
See a4830525c0aecb58839f9fe9c5173903dfebdcbc